workflow-templates/command-rebase.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+on:
+  issue_comment:
+    types: [ created ]
+name: Automatic Rebase
+
+jobs:
+  rebase:
+    name: Rebase
+    # On pull requests and if the comment starts with `/compile`
+    if: github.event.issue.pull_request != '' && startsWith(github.event.comment.body, '/rebase')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      
+      - name: Automatic Rebase
+        uses: cirrus-actions/rebase@1.3.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The only issue with that is that github-action does not trigger CI.
We might have to think a bit more about this (closing/opening the pr will trigger ci again)

Reply:
I found this: https://github.com/cirrus-actions/rebase
![image](https://user-images.githubusercontent.com/42591237/123422248-384cec80-d5be-11eb-9cbd-5c48c24b2879.png)

Reply:
@skjnldsv what do you think?

Reply:
@skjnldsv if we don't find a better solution in the short term, can we merge this and improve it later?

Reply:
This needs to be discussed with @LukasReschke :)
This requires simple standard write + workflow permissions:

Reply:
![image](https://user-images.githubusercontent.com/14975046/123430175-27a17400-d5c8-11eb-9fc5-20da196fcacc.png)

Reply:
I'm not a super big fan of adding a personal access token if we can avoid it. This usually means anyone in the org could access the token 🙈 (if we go that way, at least make it a dedicated service account in the organization)

That said: do we need "workflow permissions" even? I believe that is only needed to update a workflow file not to execute one?

I'd really like the inverse: The GitHub action would trigger a webhook and then some bot somewhere invokes the GitHub workflow with parameters using the REST API. (I believe one can parameterise this - which is what we also did on the Docker repository) Then at least one can't easily exfiltrate a token.

Reply:
The issue is having to run the bot and maintain it. They're usually not that easy and this is what we've been doing. But also what have been causing issues.
Right, I guess only having write to the repo is fine as a permission, as long as you can push some code.
What is the worst that could happen btw? With a leaked token? People that could already exfiltrate the token by creating an action already have write access to the org